includes/admin/views/html-admin-page-addons.php

```diff
@@ -77,20 +77,20 @@ class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
 				</div>
 			<?php endif; ?>
 			<?php if ( '_featured' !== $current_section && $addons ) : ?>
-				<?php if ( 'shipping_methods' === $current_section ) : ?>
+				<?php if ( 'shipping-methods' === $current_section ) : ?>
 					<div class="addons-shipping-methods">
 						<?php WC_Admin_Addons::output_wcs_banner_block(); ?>
 					</div>
 				<?php endif; ?>
 				<?php if ( 'payment-gateways' === $current_section ) : ?>
-					<div class="addons-shipping-methods">
+					<div class="addons-gateways">
```

Comment:
It's a bit confusing, but the original `addons-shipping-methods` was intentional: we have this style in `admin.scss` which applies to this bit:

```scss
	.addons-shipping-methods .addons-wcs-banner-block {
		margin-left: 0;
		margin-right: 0;
		margin-top: 1em;
	}
```

We could leave it as it is, or we could change this to `addons-payment` and tweak the CSS rule to take the new class into account.

Reply:
That makes sense, I adjusted the class name to `addons-payment` and updated `admin.scss`.